src/STAT_FrontEnd.C

```diff
@@ -1908,8 +1507,8 @@ StatError_t STAT_FrontEnd::createDaemonRankMap()
     /* First, create a map containing all daemons, with the MPI ranks that each daemon debugs */
     for (i = 0; i < nApplProcs_; i++)
     {
-        currentNode = proctab_[i].pd.host_name;
```

Comment:
virtualized to accommodate a CTI managing the proc table differently.   Here the lmon impl:

const char* STAT_lmonFrontEnd::getHostnameForProc(int procIdx)
{
    return proctab_[procIdx].pd.host_name;
}